.ci/kitchen-windows2016-py3

```diff
@@ -10,7 +10,7 @@ runTestSuite(
     nox_env_name: 'runtests-zeromq',
     nox_passthrough_opts: '--unit',
     python_version: 'py3',
-    testrun_timeout: 9,
+    testrun_timeout: 10,
```

Comment:
windows 2019 is the only one that is consitantly timing out, not 2016

Reply:
Ok, I'll reduce the scope of the change.

Reply:
Done